### DISCUSSION TITLE
Kit seems to be stuck open

### DISCUSSION THREAD

**Discussion by benjaminmodayil** [johnlindquist/kit#366] [Errors] [answered]
Kit seems to be stuck open when running the launch command and clicking outside of it or the same command to launch it doesn't close it.

It used to do this occasionally for me in the past, but now it happens every time I open it. I have to run a script and let that process for it to close properly.

**Comment by johnlindquist** [comment, marked as answer]
🤔 thanks for reporting.

Any chance youre using a script that has "ignoreBlur" set? Or using the "drop" component?

Just curious if it's ignoreBlur getting stuck or something else. I'll investigate and get it sorted out

**Reply by benjaminmodayil** [reply]
I'm not using the `drop` component. I did use a script earlier using `ignoreBlur` within my first Kit session. 

I managed to get behavior to go back to normal by quitting Kit after posting this (should have tried sooner 🤦‍♂️), but at some point it managed to get stuck open again. I quit once more and it hasn't happened yet. 

I'll follow up here if I can get it to happen again.

**Reply by johnlindquist** [reply]
Thanks for following up. It sounds like "ignoreBlur" isn't clearing properly since quitting/restarting fixes it. "ignoreBlur" is used in drop/textarea/editor and some other places. I'll make edtra sure it gets cleared out and that should resolve this.

**Reply by johnlindquist** [reply]
beta.34 has improved "ignoreBlur" handling.

Also, pressing the main keyboard shortcut while the prompt is open will close the prompt. Thanks for the idea! 😊

**Reply by benjaminmodayil** [reply]
I believe I'm on the beta.34 and have been toggling the app with the main keyboard shortcut. Looks great! Thanks! I haven't had any issues where it hangs open so I believe we're good 👍 I'll follow-up if otherwise. 

I'll mark this thread as an answer to my initial post.
